distribution/solr-distro/pom.xml

```diff
@@ -99,12 +99,13 @@
                         </goals>
                         <configuration>
                             <url>
-                                http://artifacts.codice.org/content/repositories/thirdparty/org/apache/archive/solr/${solr.version}/solr-${solr.version}.zip
+                              ${solr.download.url}/lucene/solr/${solr.version}/solr-${solr.version}.zip
                             </url>
                             <unpack>true</unpack>
                             <failOnError>true</failOnError>
-                            <sha512>aca27ff6942872a571160f49f5e4d43090dbe9bb37da0d1999557e1655f25055173ef999bfb8eba8f19f19b9c438b43205c805d0d294b9756cf5c0aaf66a656a</sha512>
-                            <checkSignature>true</checkSignature>
+                            <!--temporarily disable check so the zip can be obtained from multiple sources 
```

Comment:
The SHA512 checksum for Solr 7.7.2 should always be the same regardless of source.

:question: Should the SHA512 be a property too?

Reply:
Looks like it works now I was probably doing something else wrong so removed this section.